### PR TITLE
Pushes the Aurora event into meme team submarine territory.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: Guyonbroadway
tweak: SUPERINTENDENT!! I Was just... uhh... changing the audiofiles for a tg event! Memetastic sound triggers! Care to laugh with me? 
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

### CODE REVIEW

**Comment by Jujumatic** [comment]
Yes but also change the description of the event so it's memeified as well

**Review by JohnGinnane — APPROVED** [review]

File: hippiestation/code/modules/events/aurora_caelus.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+/datum/round_event/aurora_caelus/announce()
+	priority_announce("[station_name()]: A harmless cloud of ions is approaching your station, and will exhaust their energy battering the hull. Nanotrasen has approved a short break for all employees to relax and observe this very rare event. During this time, starlight will be bright but gentle, shifting between quiet green and blue colors. Any staff who would like to view these lights for themselves may proceed to the area nearest to them with viewing ports to open space. We hope you enjoy the lights.",
+	sound = 'sound/misc/notice2.ogg',
+	sender_override = "Nanotrasen Meteorology Division")
+	for(var/V in GLOB.player_list)
+		var/mob/M = V
+		if((M.client.prefs.toggles & SOUND_MIDI) && is_station_level(M.z))
+			M.playsound_local(M, 'hippiestation/sound/ambience/auroraborealis.ogg', 20, FALSE, pressure_affected = FALSE)
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/events/aurora_caelus.dm:9]: Gonna be totally honest, there should be an option to disable the music. It's really annoying after a while.

Even then this event is poop. "CHECK OUT THE HULL, GREEN LIGHTS. WHOAAAAAAAAAA. PLAY MUSIC FOR FUCKING EVERYONE"

but yeah whatever steamed hams

**Review by Manly-Dwarf — APPROVED** [review]

File: hippiestation/code/modules/events/aurora_caelus.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/events/aurora_caelus.dm:9]: `if((M.client.prefs.toggles & SOUND_MIDI) ... )`
Looks to be optional to me, also the clientside stopsounds verb should cancel it.

**Comment by praisenarsie** [comment]
the event is shit and garbage

**Comment by Carbonhell** [comment]
huh,sure +1

**Comment by ExpAntique** [comment]
please. The original event is snowflake garbage, and has no place on hippie's 2000 iq codebase

**Comment by earthmoonjune** [comment]
+1

**Comment by Jujumatic** [comment]
does this still work

**Pull request merged**